pkgs/development/compilers/graalvm/graalvm-oracle/hashes.nix

```
@@ -0,0 +1,38 @@
+{
+  "22" = {
+    "aarch64-linux" = {
+      sha256 = "sha256-0wPTDBB2T+qrB27+eQGd88mBsEh/ysPfelMT929hBA4=";
```

```suggestion
      hash = "sha256-0wPTDBB2T+qrB27+eQGd88mBsEh/ysPfelMT929hBA4=";
```

(and the same below)